### PR TITLE
Add nachos to favourite foods and command line basics to next time suggestions

### PR DESCRIPTION
**Pull request by JesseWilliamson**

### CODE REVIEW

**Review by PiiDish — DISMISSED** [review]
LGTM

**Comment by EthanBirdB** [comment]
nachos are cool and all, but they're no bread

**Comment by ronjain99** [comment]
Yeah good shout

File: Foods.yaml [hunk 1]
```diff
@@ -6,5 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
```
Comment (ronjain99) [inline Foods.yaml:8]: cool story bro

**Review by EthanBirdB — APPROVED** [review]
Might have to do this one more time whoops

**Review by ronjain99 — COMMENTED** [review]
Aight easy

**Pull request merged**